ghost/i18n/locales/sk/comments.json

```
@@ -55,16 +55,18 @@
     "Sending": "Odosielanie",
     "Sent": "Odoslané",
     "Show": "Zobraziť",
+    "Show {{amount}} more replies": "Zobraziť {{amount}} ďalších odpovedí",
+    "Show {{amount}} previous comments": "Zobraziť {{amount}} predchádzajúcich komentárov",
```

Biome (1.9.4)

[error] The key Show {{amount}} more replies was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)